starter/src/test/java/com/starter/fullstack/rest/InventoryControllerTest.java

```diff
@@ -64,4 +65,15 @@ public void create() throws Throwable {
 
     Assert.assertEquals(2, this.mongoTemplate.findAll(Inventory.class).size());
   }
+
+  @Test
+  public void remove() throws Throwable {
+    this.mockMvc.perform(delete("/inventory")
+        .accept(MediaType.APPLICATION_JSON)
+        .contentType(MediaType.APPLICATION_JSON)
+        .content("[\"" + this.inventory.getId() + "\"]"))
```

Comment:
I know product does this, but the better approach is actually to do List.of(this.inventory.getId()).toString()